### PR TITLE
Refactor to a monorepo.

### PR DESCRIPTION
**Pull request by mdwelsh**
This moves the web app code into packages/webapp and brings in TurboRepo so we can build multiple packages in the same repo if needed. (The first one will likely be the tool to ingest a new podcast.)

### CODE REVIEW

File: packages/webapp/src/app/layout.tsx [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
   description: 'Generated by create next app',
-}
+};
 
-export default function RootLayout({
-  children,
-}: {
-  children: React.ReactNode
-}) {
+export default function RootLayout({ children }: { children: React.ReactNode }) {
```
Comment (farzadab) [inline packages/webapp/src/app/layout.tsx:11]: Q: what does this do?
I wanted to create a parent component for all the pages. For example the ThemeProvider would live there.
Can this `RootLayout` thing do that?

Comment (farzadab) [reply packages/webapp/src/app/layout.tsx:11]: If yes, I'd like to push the `AppHeader` into the RootLayout too.

**Review by farzadab — APPROVED** [review]
LGTM.
I had a random NextJS question though.

File: packages/webapp/src/app/layout.tsx (hunk 1 above)
Comment (mdwelsh) [reply packages/webapp/src/app/layout.tsx:11]: TBH I don't really know for sure. I think it is the main page layout for all pages in the app. Check the Next.js docs :-)

**Pull request merged**